kolibri/core/content/management/commands/deletecontent.py

```diff
@@ -138,13 +149,10 @@ def handle_async(self, *args, **options):
                 "file_size__sum", 0
             )
             job.extra_metadata["file_size"] = total_file_deletion_size
-            job.extra_metadata["total_resources"] = total_file_deletion_operations
+            job.extra_metadata["total_resources"] = total_resource_number
             job.save_meta()
-
```

Comment:
Would be better to avoid flyby commits like this - seems like your IDE might be reformatting files when you save?

Reply:
I do not have auto reformatting on files when i save but i did reformatting. I wont use reformatting anymore on whole file.